### DISCUSSION TITLE
I suggest the possibility in the team talk program to broadcast without internet or with very low and almost absent connections.

### DISCUSSION THREAD

**Discussion by alessio0292** [BearWare/TeamTalk5#1239] [Ideas]
Dear Bjoern and all,
I suggest the possibility of broadcasting offline for example in an offline channel so, in this way, there will be no annoying delays or interruptions that are the cause of the internet connections and people would not worry about these problems.
I don't know but I think such a thing can be done.
Greetings,
Alessio from Italy.
p.s .: The possibility of creating a server can also be useful and also the possibility of creating servers that work even without internet.
in practice, example, a person with only a sound card that have the program
can use me also, without a internet connection I can have a connection but
if a connection is very low or doesn’t run or work, I can same use a program
and i can broadcast and transmit same very well using only a sound card or
for mobile users, the cards of phones.

**Comment by tech10** [comment]
I fail to see how this would work, an internet connection is required for two people to talk over any voice protocol with two computers, especially when you're dealing with mobile devices. Two people with two computers can't communicate with only their sound cards, unless they had very long cables to connect their sound cards together, which would only work for two people, and only if they were in sufficient range for such a thing. As far as TeamTalk goes, you need a network connection, whether it be local or over the internet, to communicate with anyone on a server, there couldn't be such a thing as an off line channel that would work with no internet connection.

**Comment by alessio0292** [comment]
For me there might be a solution for use team talk offline then I don’t 
know.
Unfortunately, The problems in internet are for example the audio 
interruptions, the interval transmission and other things that I don’t like 
but those that i said are the importants that give problems in team talk.

Da: Blake G
Data: mercoledì 23 febbraio 2022 22:31
A: BearWare/TeamTalk5
Cc: alessio0292 ; Author
Oggetto: Re: [BearWare/TeamTalk5] I suggest the possibility in the team talk 
program to broadcast without internet or with very low and almost absent 
connections. (Discussion #1239)

I fail to see how this would work, an internet connection is required for 
two people to talk over any voice protocol with two computers, especially 
when you're dealing with mobile devices. Two people with two computers can't 
communicate with only their sound cards, unless they had very long cables to 
connect their sound cards together, which would only work for two people, 
and only if they were in sufficient range for such a thing. As far as 
TeamTalk goes, you need a network connection, whether it be local or over 
the internet, to communicate with anyone on a server, there couldn't be such 
a thing as an off line channel that would work with no internet connection.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
You are receiving this because you authored the thread.Message ID: 
***@***.***>

**Comment by tech10** [comment]
I don't think what you're proposing is possible. Also, there is no such thing as a server without some sort of network connection. Otherwise, it isn't a server. It's a self-contained program that can't access anything else, that's what TeamTalk would be if there was no internet or network connection. You couldn't communicate with anyone else.

**Reply by alessio0292** [reply]
but in this way I don’t like team talk in this moment I’d like best things
because i don’t like and want the current things!
excuse me for my english because i’m italian.

**Comment by alessio0292** [comment]
but in this way I don’t like team talk in this moment I’d like best things 
because i don’t the current things!
excuse me for my english because i’m italian.
Da: Blake G
Data: mercoledì 23 febbraio 2022 22:51
A: BearWare/TeamTalk5
Cc: alessio0292 ; Author
Oggetto: Re: [BearWare/TeamTalk5] I suggest the possibility in the team talk 
program to broadcast without internet or with very low and almost absent 
connections. (Discussion #1239)

I don't think what you're proposing is possible. Also, there is no such 
thing as a server without some sort of network connection. Otherwise, it 
isn't a server. It's a self-contained program that can't access anything 
else, that's what TeamTalk would be if there was no internet or network 
connection. You couldn't communicate with anyone else.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
You are receiving this because you authored the thread.Message ID: 
***@***.***>

**Comment by ChrisDuffley** [comment]
I can't see how it would be possible either. There's no way to do this--at least not with the way that the SDK works and things like that. I'm sorry.

Christopher Duffley

Host of Mission Possible Podcast <http://www.missionpossiblepodcast.com/>

LINKS TO SUBSCRIBE:

iTunes <https://itunes.apple.com/us/podcast/mission-possible-christopher/id1103092109?mt=2> | Stitcher <http://www.stitcher.com/s?fid=88094&refid=stpr> | Google Play <https://goo.gl/app/playmusic?ibi=com.google.PlayMusic&isi=691797987&ius=googleplaymusic&link=https://play.google.com/music/m/Ik76dexklcprwnpuxbdr2s6bm3y?t=Mission_Possible_with_Christopher_Duffley> | Follow Podcast On TuneIn And Listen To Latest Episode <http://tunein.com/radio/Mission-Possible-With-Christopher-Duffley-p892505/>

LIKE/FOLLOW Me:

Facebook <http://facebook.com/christopherduffley> | Twitter <http://twitter.com/chrisduffley> | YouTube <http://youtube.com/christopherduffley> | Instagram <http://instagram.com/christopherduffley>

On Feb 23, 2022, at 16:51, Blake G ***@***.***> wrote:


I don't think what you're proposing is possible. Also, there is no such thing as a server without some sort of network connection. Otherwise, it isn't a server. It's a self-contained program that can't access anything else, that's what TeamTalk would be if there was no internet or network connection. You couldn't communicate with anyone else.

**Comment by ChrisDuffley** [comment]
Get a peer to peer app instead. TeamTalk is not this way at this time, but it's better at lower capacities and as long as things are close to each other physically the delay will be much, much less.

Christopher Duffley

Host of Mission Possible Podcast <http://www.missionpossiblepodcast.com/>

LINKS TO SUBSCRIBE:

iTunes <https://itunes.apple.com/us/podcast/mission-possible-christopher/id1103092109?mt=2> | Stitcher <http://www.stitcher.com/s?fid=88094&refid=stpr> | Google Play <https://goo.gl/app/playmusic?ibi=com.google.PlayMusic&isi=691797987&ius=googleplaymusic&link=https://play.google.com/music/m/Ik76dexklcprwnpuxbdr2s6bm3y?t=Mission_Possible_with_Christopher_Duffley> | Follow Podcast On TuneIn And Listen To Latest Episode <http://tunein.com/radio/Mission-Possible-With-Christopher-Duffley-p892505/>

LIKE/FOLLOW Me:

Facebook <http://facebook.com/christopherduffley> | Twitter <http://twitter.com/chrisduffley> | YouTube <http://youtube.com/christopherduffley> | Instagram <http://instagram.com/christopherduffley>

On Feb 23, 2022, at 16:55, alessio0292 ***@***.***> wrote:


but in this way I don’t like team talk in this moment I’d like best things 
because i don’t the current things!
excuse me for my english because i’m italian.
Da: Blake G
Data: mercoledì 23 febbraio 2022 22:51
A: BearWare/TeamTalk5
Cc: alessio0292 ; Author
Oggetto: Re: [BearWare/TeamTalk5] I suggest the possibility in the team talk 
program to broadcast without internet or with very low and almost absent 
connections. (Discussion #1239)

I don't think what you're proposing is possible. Also, there is no such 
thing as a server without some sort of network connection. Otherwise, it 
isn't a server. It's a self-contained program that can't access anything 
else, that's what TeamTalk would be if there was no internet or network 
connection. You couldn't communicate with anyone else.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
You are receiving this because you authored the thread.Message ID: 
***@***.***>

**Comment by tech10** [comment]
A peer to peer application would only lower latency between two people. Any more than that and you'd run into other problems, such as each person requiring that they transmit to every other person connected, which would eventually take more bandwidth than TeamTalk does. Also, it wouldn't eliminate any connection problems or other issues that come with all internet or network connections, eventually, it's unavoidable, especially in today's world where everything is increasingly being done over the internet.

**Comment by alessio0292** [comment]
I guys don't know what to do I don't know any better program than team talk. 
I would like the team talk program to improve.

Da: Blake G
Data: mercoledì 23 febbraio 2022 23:08
A: BearWare/TeamTalk5
Cc: alessio0292 ; Author
Oggetto: Re: [BearWare/TeamTalk5] I suggest the possibility in the team talk 
program to broadcast without internet or with very low and almost absent 
connections. (Discussion #1239)

A peer to peer application would only lower latency between two people. Any 
more than that and you'd run into other problems, such as each person 
requiring that they transmit to every other person connected, which would 
eventually take more bandwidth than TeamTalk does. Also, it wouldn't 
eliminate any connection problems or other issues that come with all 
internet or network connections, eventually, it's unavoidable, especially in 
today's world where everything is increasingly being done over the internet.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
You are receiving this because you authored the thread.Message ID: 
***@***.***>

**Comment by alessio0292** [comment]
please @bear101, read this discussion because in this moment i hate team talk! I ate hate very much!

**Comment by alessio0292** [comment]
in this moment @tech10 team talk is a shit, i hate very much team talk i did many discussion and Bjoern doesn't want things for me! He doesn't want to do nothing for me! @bear101 also read this discussion and others that i did for you and other programmers no italian because all, You aren't italians!

**Comment by tech10** [comment]
@alessio0292 As you yourself stated, delays and connections not being entirely stable is a reality of the internet. This is the case when dealing with any voice or video transmission protocol. There isn't one that will be entirely stable as you're hoping to achieve. What you want to achieve over the internet isn't something that can be done with a live chatting system. If you don't want any delay in any audio transmitted over the internet, record an audio file and send it to someone, who can then play it on their own computer at their own time, then send you one back. Of course, this eliminates a system of live chat. There can't be an off line channel on TeamTalk that uses soundcard connections, there is no way to do such a thing to talk with someone else in another location. You need some type of a network connection for this, as I've stated before. I would personally find your suggestions to create a stable connection on TeamTalk over the internet an impractical exercise. As a final point, whether or not someone is Italian has nothing to do with an internet connection, it won't change the realities of online communication.

**Reply by alessio0292** [reply]
Oh shit! No because i'd like speak in my language i don't like speak en english but this software is not italian! Internet shit, I hate very much team talk because I did also others discussion because there are many things of this program that I don't like! Is since 2012 that i use this shit dear @tech10 understand?

**Reply by tech10** [reply]
If you don't like it, don't use it. It's that simple. Until there's any productive discussion here, I'm not going to respond further.

**Comment by alessio0292** [comment]
hey also for @ChrisDuffley, I did many discussions for Bjoern and You but there aren't improvemtens for nothing for things that i suggest! I always more hate team talk unfortunately this program is a shit excuse me but i'm not happy for this program!

**Comment by alessio0292** [comment]
hey also for @ChrisDuffley, I did many discussions for Bjoern and You but there aren't improvemens for nothing for things that i suggest! I always more hate team talk unfortunately this program is a shit excuse me but i'm not happy for this program!

**Comment by alessio0292** [comment]
hey also for @ChrisDuffley, I did many discussions for Bjoern and You but there aren't improvements for nothing for things that i suggest! I always more hate team talk unfortunately this program is a shit excuse me but i'm not happy for this program!

**Comment by computertech1012** [comment]
Hi  everyone, the only way that you would be able to use TeamTalk without Internet, would be if you have a router set up, and are on the same network, you could then connect to TeamTalk on your local Netwerk. That is the only way that you will be able to use this program off-line. In this way, if you are using an ethernet connection or Wi-Fi, you could use it as a home system in this way, but this would be the only way that you would be able to achieve what you were trying to achieve.

**Comment by alessio0292** [comment]
yes but i’d like speak with the world for example Brazil also without 
network connection for example!
Alessio italian.
Da: computertech1012
Data: giovedì 24 marzo 2022 01:17
A: BearWare/TeamTalk5
Cc: alessio0292 ; Mention
Oggetto: Re: [BearWare/TeamTalk5] I suggest the possibility in the team talk 
program to broadcast without internet or with very low and almost absent 
connections. (Discussion #1239)

Hi everyone, the only way that you would be able to use TeamTalk without 
Internet, would be if you have a router set up, and are on the same network, 
you could then connect to TeamTalk on your local Netwerk. That is the only 
way that you will be able to use this program off-line. In this way, if you 
are using an ethernet connection or Wi-Fi, you could use it as a home system 
in this way, but this would be the only way that you would be able to 
achieve what you were trying to achieve.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: 
***@***.***>

**Comment by alessio0292** [comment]
hey also @CoBC pleas read also this discussion and others that i did because in this bad program I'm not fine more and in this moment i uninstalled it because from many years always very more I'm hating this program for many things that i don't like and I'm proposing various suggestion and improvements that I written in many discussion from 1th november 2021!

**Comment by alessio0292** [comment]
yes but i’d like speak with the world for example Brazil also without
network connection for example! do you understand @computertech1012?
Alessio italian.

**Reply by computertech1012** [reply]
I think I understand, you kind of want something that's similar to a phone conversation, that doesn't require Internet, right? This would be a neat feature. Something that goes off of a cellular network for example, something that would be similar to a phone conversation. Or, something that would be related to for example, a radio or a ham radio with towers, that doesn't require network connection. Am I understanding you correctly on that one?

**Reply by alessio0292** [reply]
I'd like that the people speak only with a sound card for example download the app in internet and then can also speak with people without internet and always with audio hi-fi!

**Reply by alessio0292** [reply]
Hey @computertech1012, I don't know if you are american or english, but, You can't speak or write en english I think that You don't know the english for this thing, write in italian or your language! Neat is a incorrect word the word is net! You are not american or english for me speak in my language, italian!

**Comment by alessio0292** [comment]
hey @bear101 read this discussion and others that i written because now i uninstalled this shit of this program!

**Comment by alessio0292** [comment]
Guys I want to continue this discussion and still invite the TeamTalk master to read this discussion.

**Comment by alessio0292** [comment]
please other master @CoBC from France, read also this discussion because is important for the future of team talk and also i repeat, master owner @bear101, read this discussion.

**Comment by alessio0292** [comment]
i'm very worried because Bjoern doesn't consider my requests and also a french @CoBC

**Comment by alessio0292** [comment]
I don’t undestand your english only abith @computer tech. Please italian or 
I don’t know.
I’d like to use team talk without internet or net connection also in a 
computer.

Da: computertech1012
Data: sabato 26 marzo 2022 00:22
A: BearWare/TeamTalk5
Cc: alessio0292 ; Mention
Oggetto: Re: [BearWare/TeamTalk5] I suggest the possibility in the team talk 
program to broadcast without internet or with very low and almost absent 
connections. (Discussion #1239)

I think I understand, you kind of want something that's similar to a phone 
conversation, that doesn't require Internet, right? This would be a neat 
feature. Something that goes off of a cellular network for example, 
something that would be similar to a phone conversation. Or, something that 
would be related to for example, a radio or a ham radio with towers, that 
doesn't require network connection. Am I understanding you correctly on that 
one?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: 
***@***.***>
